src/mtool/ui/src/components/VolumeManagement/AdvanceCreateVolume/index.js

```diff
@@ -886,7 +886,7 @@ function AdvanceCreateVolume(props) {
         {props.volume_count > 1 &&
           <Typography className={classes.previewElement}>Start Suffix Value : {props.volume_suffix}</Typography>
         }
-        {props.volume_size === 0 ? (
+        {parseInt(props.volume_size, 10) === 0 ? (
```

Comment:
The volume field also works with floating point values. So, please use parseFloat instead of parseInt

Reply:
Changed to parseFloat  thanks for the comment.